lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -823,15 +860,15 @@ mapping:
 
       instance_resource_url:
         type: str
-        default: https://galaxyproject.org/
         required: false
         desc: |
           URL of the support resource for the galaxy instance.  Used in activation
           emails.
 
+          Sample default 'https://galaxyproject.org/'
```

Comment:
```suggestion
          Example value 'https://galaxyproject.org/'
```

Reply:
Sounds good, and I assume you meant this for all instances of 'Sample default' (correct me if you meant just this one)